app/support/notification_sender.rb

```diff
@@ -2,36 +2,31 @@
 
 class NotificationSender
```

Comment:
While we're in here, can we try moving this from the `app/support` folder into the `app/services` folder? It's not a pressing need, but I have a desire to eventually be rid of the `app/support` directory. It manifests itself in the confusing `spec/support` (supporting rspec files like configuration and shared contexts) and `spec/app_support` (where we test the `app/support` classes.

So then if this moves, the affiliated spec should move as well.

Reply:
Oh fun - I 💯 support this

Reply:
I forget how we ended up in this scenario, but I'm pretty sure it's my fault--at least for the spec folder. There's also a lot of stuff in `lib` that probably belongs in `app`, but it's been kind of a move it as we touch it kind of thing.